client/src/components/PaginationController.tsx

```diff
@@ -0,0 +1,87 @@
+import { MouseEventHandler } from 'react';
+import styled from 'styled-components';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro';
+
+interface PaginationControllerProps {
+  paginationRange: number[];
+  currentPage: number;
+  setCurrentPage: Function;
+}
+
+const PaginationController = ({
+  paginationRange,
+  currentPage,
+  setCurrentPage,
+}: PaginationControllerProps) => {
+  // If there are less than 2 times in pagination range we shall not render the component
+  if (currentPage === 0 || paginationRange.length < 2) {
+    return null;
+  }
+
+  const firstPage: number = paginationRange[0];
+  const leftArrowActive: boolean = currentPage !== firstPage;
+  const lastPage: number = paginationRange[paginationRange.length - 1];
+  const rightArrowActive: boolean = currentPage !== lastPage;
+
+  return (
+    <PaginationContainer>
+      <PaginationDiv
+        onClick={() => {
+          if (leftArrowActive) setCurrentPage(currentPage - 1);
```

Comment:
what happens if user is already on first page?
is this code reachable? or is the left arrow already greyed out?